cristin-project/build.gradle

```diff
@@ -16,30 +17,8 @@ dependencies {
     testImplementation project(":cristin-testing")
 }
 
-
-tasks.named('test') {
+tasks.named('test', Test) {
     useJUnitPlatform {
         excludeTags 'karateTest'
     }
-    failFast = true
-    testLogging {
-        events 'skipped', 'passed', 'failed'
-    }
-    finalizedBy jacocoTestReport
 }
-
-sourceSets {
-    test {
-        resources {
-            srcDir file('src/test/java')
-            exclude '**/*.java'
-        }
-    }
-}
-
-test{
-    environment "COGNITO_URI", "https://example.cognito.org"
-    environment "CRISTIN_BOT_FILTER_BYPASS_HEADER_NAME", "Hello"
-    environment "CRISTIN_BOT_FILTER_BYPASS_HEADER_VALUE", "World"
-}
```

Comment:
You don't need to keep these anymore?

Reply:
I moved them to nva.cristin.service.core.java-conventions.gradle task -> test
(as this applies to every subtest (no need to repeat information))